docs/source/guides/tools_menu.rst

```diff
@@ -196,6 +196,14 @@ Events Run On
 
 This function is also used to select the core that runs the Arduino events. This is only valid if the target SoC has 2 cores.
 
+Erase Flash
+***********
+
+This function is used to select if the flash memory should be erased before uploading the sketch.
```

Comment:
Maybe like this? "This function enables the flash memory region to be erased before uploading the new sketch."

Reply:
Changed :)